src/pages/dashboard/components/density-preferences/images/comfortable-visual-refresh.tsx

```diff
@@ -4,15 +4,15 @@ import React from 'react';
 
 import { TableRow, TableRows, TopNavigation, WindowPath } from './common';
 
-import styles from './styles.module.scss';
+import * as styles from './styles.module.scss';
```

Comment:
Why is it now necessary to import with `* as`?

Reply:
I see, this is coming from the css-loader upgrade: https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#700-2024-04-04